src/server/controller/cycleData/persistNodeValue/getDependants.ts

```diff
@@ -0,0 +1,52 @@
+import { CountryIso } from '@meta/area'
+import { Assessment, Cycle, TableNames } from '@meta/assessment'
+
+import { AreaController } from '@server/controller/area'
+import { isODPVariable } from '@server/controller/cycleData/originalDataPointVariables'
+import { OriginalDataPointRepository } from '@server/repository/assessmentCycle/originalDataPoint'
+
+type Props = {
+  assessment: Assessment
+  countryIso: CountryIso
+  cycle: Cycle
+  tableName: string
+  variableName: string
+  colName: string
+  isODP?: boolean
+}
+
+export const isODPCell = async (props: Props) => {
+  const { colName, tableName } = props
+  const country = await AreaController.getCountry(props)
+  const odpYears = await OriginalDataPointRepository.getReservedYears(props)
+  if (![TableNames.forestCharacteristics, TableNames.extentOfForest].includes(tableName as TableNames)) return false
+  return country.props.forestCharacteristics.useOriginalDataPoint && odpYears.map(String).includes(colName)
+}
+
+// Case1:
+// If editing ODP,
+// update all dependants found in metaCache
+// but exclude odp variables
+
+// Case2:
+// If table is 1a or 1b
+// and if there is an ODP year
+// and if country.odp = enabled
+// we are editing 'under' odp
```

Comment:
can we improve this comment like Case1, explicitly mentioning which dependants get updated? (see reference issue)